db/migrate/20220920110211_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Please follow the ERD provided for this tutorial, right now you are not affecting the Author as a reference for the user.